### ISSUE TITLE
Uncaught TypeError: _a.start is not a function

### ISSUE THREAD

**Issue by productdevbook** [fatihsolhan/v-onboarding#3]
<img width="717" alt="CleanShot 2021-12-20 at 08 23 02@2x" src="https://user-images.githubusercontent.com/38668796/146715841-57d48698-7274-4610-97d7-3f09182bb336.png">

```
TypeError: Cannot destructure property 'previous' of 'undefined' as it is undefined.
```

**Comment by fatihsolhan** [comment]
Should be fixed in 1.0.1 @productdevbook 
Can you please check if the issue still exists?

**Comment by productdevbook** [comment]
dont fixed

onyly see
import {OnboardingWrapper, } from 'v-onboarding'

**Comment by fatihsolhan** [comment]
> dont fixed
> 
> onyly see import {OnboardingWrapper, } from 'v-onboarding'

This was a typing issue, fixed in #8
